src/languages/postgresql/postgresql.formatter.ts

```diff
@@ -311,11 +292,40 @@ const binaryOperators = [
   '#>',
   '#>>',
   '#-',
-  // Other Operators
-  ':=',
-  '::',
+  // Named function params
   '=>',
+  // Network address
+  '>>=',
+  '<<=',
+  // Pattern matching
+  '~~',
+  '~~*',
+  '!~~',
+  '!~~*',
+  // POSIX RegExp
+  '~',
+  '~*',
+  '!~',
+  '!~*',
+  // Range/multirange
   '-|-',
+  // Similarity (TODO: not in wiki)
+  '<%',
+  '<<%',
+  '%>',
+  '%>>',
+  // Byte comparison (TODO: not in wiki)
+  '~>~',
+  '~<~',
+  '~>=~',
+  '~<=~',
```

Comment:
These were added in #296, but I'm unable to locate documentation for these.

@inferrinizzard do you perhaps know where I could locate these in PostgreSQL docs?

Reply:
the `~_~` operators are not in the docs, found from this SO link: https://stackoverflow.com/a/35808554/10028067
if you run `\do ~>~` or similar in pg cli, you get these results:
![image](https://user-images.githubusercontent.com/10782902/188332052-5ca44105-b599-433d-a596-5d1f4cc80dda.png)

Reply:
the `%` ish operators are in table F.26 here: https://www.postgresql.org/docs/current/pgtrgm.html#id-1.11.7.42.6

Reply:
it seems there are more operators I missed:
```
<->
<<->
<->>
<<<->
<->>>
```